src/app/components/dropdown/index.html

```diff
@@ -1,15 +1,176 @@
-<sky-demo-page
-  pageTitle="Dropdown"
-  skyuxModule="SkyDropdownModule"
-  packageName="@skyux/popovers"
+<sky-tabbed-demo-page
+  pageTitle="Dropodown"
 >
 
-  <sky-demo-page-summary>
-    <p>
-      The dropdown component renders a button that displays a menu when users click it. The menu displays a set of actions or functions. <strong>Do not use the dropdown component as a select input</strong> for users to set the menu's value to one of the options. If you need a select input, see the <a routerLink="/design/guidelines/select-items">options to select items from lists</a>.
-    </p>
-  </sky-demo-page-summary>
+<sky-demo-page-summary>
+  <p>
+    The dropdown component renders a button that displays a menu when users click it. The menu displays a set of actions or functions.
+  </p>
+</sky-demo-page-summary>  
 
+<sky-demo-page-example>
+  <sky-dropdown-demo>
+  </sky-dropdown-demo>
+</sky-demo-page-example>  
+  
+<sky-tabset id="content-tabs">
+
+  <sky-tab
+    tabHeading="Design">
+    
+    <sky-demo-page-content sectionHeading="Usage">
+      <h3 class="sky-text-success">Use when</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Use dropdowns when users require access to multiple, related actions or when users require access to multiple, different actions and there is not enough space to use separate buttons for each action.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/dropdown/usage-toolbar.png" />
+            <div class="caption caption-success sky-text-success">Do use dropdowns to let users access multiple actions.</div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Use context menu dropdowns for actions that affect one item in a list.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <img src="~/assets/img/guidelines/dropdown/usage-context-menu.png" />
+            <div class="caption caption-success sky-text-success">Do use context menu dropdowns with items in a list.</div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Use icon-only dropdowns when there is not enough space to include a text label, such as in a list toolbar at phone resolutions.</p>
```

Comment:
The such as clause is a misplaced modifier and makes this read as a run-on sentence. I'm going to reword slightly and break this into two sentences.

Reply:
Also, is "at phone resolutions" the correct terminology? I don't think we use this term anywhere else in the docs. I think we generally refer to narrow screen widths. I'm going to reword to be consistent with that, but let me know if I'm wrong and we want to refer to phone resolutions instead of narrow widths.

Reply:
That's probably a better way of putting it.

Reply:
Good deal. I just pushed up the change along with a variety of other tweaks. I can undo any changes that don't work.